packages/hash/shared/src/entityStore.ts

```diff
@@ -39,11 +37,29 @@ export type DraftEntity<Type extends EntityStoreType = EntityStoreType> = {
   ? { properties: PropertiesType<Type["properties"]> }
   : {});
 
+type CreatedDraftEntity<Type extends EntityStoreType = EntityStoreType> = {
+  accountId: string;
+  entityId: string;
+} & DraftEntityProperties<Type>;
+
+type NotCreatedDraftEntity<Type extends EntityStoreType = EntityStoreType> = {
+  accountId: null | string;
```

Comment:
I wonder if we should just expect accountId to always be a string. We can always set it as the current user id when creating a draft entity